doc/doxygen/install/common-cmake-parameters.doxygen

```diff
@@ -59,7 +59,7 @@ The most important CMake variables are:
     (Default: On)</td>
   </tr>
   <tr>
-    <td valign="top"><nobr>\c ENABLE_UPDATE_CHECK=On/Off</nobr></td>
```

Comment:
Not unreadable but very ugly.

Reply:
I dont think this is possible in Doxygen, since it supports Latex output as well and it would break there.

What would fix it though is to modify the CSS which has
```
code {
    display: inline;
    background: var(--code-background);
    color: var(--code-foreground);
    padding: 2px 6px;
    /*! word-break: break-word; */
}
```
for `\code` sections. I have no idea why work-break would be a good idea in the first place, but disabling it fixes the problem.
Should we just do that?

Reply:
yes (as discussed offline)